contracts/PoaOperator.sol

```diff
@@ -71,10 +71,11 @@ contract PoaOperator is Adminable {
   mapping(uint256 => Slot) public slots;
 
 
-  function initialize(Bridge _bridge, Vault _vault, uint256 _epochLength) public initializer {
+  function initialize(Bridge _bridge, Vault _vault, uint256 _epochLength, uint256 _chalDuration) public initializer {
```

Comment:
use full name `challengeDuration` here and everywhere else instead of Chal ? Allows to spend less brain cycles when reading code